test/base.test.js

```diff
@@ -93,6 +94,30 @@ describe('lib.base', () => {
       });
     });
 
+    describe('.delete', () => {
+      const podName = 'pod-name';
+      const query = { pretty: true };
+      const body = {
+        apiVersion: 'v1',
+        kind: 'Pod',
```

Comment:
I think this should be `'DeleteOptions'` (it's the Kind of object you're passing in the body, not the Kind of object you want to delete)